src/triggers/sendEmail.trigger

```diff
@@ -0,0 +1,45 @@
+trigger sendEmail on Compensation__c (before insert) {
+    
+    for(Compensation__c comp: Trigger.new)
+    {
+        String htmlBody = '';
+        string subject = '';
+        Id contactId = (Id)comp.get('Contact__c');
+        Contact ct = ([SELECT Email FROM Contact WHERE Id =:contactId limit 1]);
```

Comment:
Please refrain from using SOQL queries inside loops. The way to solve this is to simply gather all ContactIds from the compensation into a list (need a for loop to do that) and then perform a SELECT from the Contact object where the Id of records retrieved is IN that list. After that, you want to loop through the records again and see which contact has to be sent an email to.